adamichelle/wayfarer-api
# #167038539 API consumer should be able to sign up a user

#### What does this PR do?
It creates the endpoint to sign up a user `POST api/v1/auth/signup`

#### Description of Task to be completed?
Have the endpoint to sign up a user working
`POST /api/v1/auth/signup`

#### How should this be manually tested?
After cloning the repo, cd into it and run `npm install`
Create a .env file containing a `DB_URL`, `PRIVATE_KEY` variable which you should set to your database's url and a string or random characters
N.B: You can generate a pair of public and private keys [here](https://www.csfieldguide.org.nz/en/interactives/rsa-key-generator/).
Run `npm run create` to create the users table in your db.
Then run `npm run dev`
Using POSTMAN test the endpoint above with this header:
_key:_ Content-Type _value:_ application/json
